Tests/Twig/Extension/SonataAdminExtensionTest.php

```diff
@@ -340,6 +342,16 @@ public function testDeprecatedRenderListElement($expected, $value, array $option
         );
     }
 
+    public function buildUrl($url)
+    {
+        if (defined('PHP_QUERY_RFC3986')) {
+            // add htmlspecialchars because twig add it auto
```

Comment:
What I don't understand is the comment : why add htmlspecialchars if twig adds it automatically?